contributions/catalog/controllers/config.py

```diff
@@ -24,3 +38,4 @@ class Config(object):
     AUTHORIZATION_BASE_URL = os.getenv("AUTHORIZATION_BASE_URL", 'https://github.com/login/oauth/authorize')
     TOKEN_URL = os.getenv("TOKEN_URL", 'https://github.com/login/oauth/access_token')
     CORS_ENABLED = bool(os.getenv('CORS_ENABLED', 'False') == 'True')
+    ADMIN_USERS = os.getenv('ADMIN_USERS', [])
```

Comment:
```suggestion
    ADMIN_USERS = os.getenv('ADMIN_USERS', '')
```
I think the default value should be the empty string as this config expects a string literal.

Reply:
I have the feedback for Contributions configs.py.

Reply:
you're right. must be confused with contributionAdmins. Thanks for catching them.